test/testhelpers.sh

```diff
@@ -148,3 +148,9 @@ shutdown() {
     fi
   fi
 }
+
+assert_file_line_count() {
+  local filename="$1"
+  local lines="$2"
+  [ $(wc -l < "$filename") -eq "$lines" ]
```

Comment:
This helper isn't saving any lines of code, and only obscures the simple unix command. I'd rather keep the tests straight forward as much as possible.